release-1.8/release_notes_draft.md

```diff
@@ -317,69 +318,65 @@ kind.
  #### Defaults
 
  - The default spec.updateStrategy for the StatefulSet and DaemonSet kinds is
- RollingUpdate for the apps/v1beta2 group version. Users may specifically set
- the OnDelete strategy, and no strategy auto-conversion will be applied to
- replace defaulted values.
+ RollingUpdate for the apps/v1beta2 group version. Users can explicitly set
+ the OnDelete strategy, and no strategy auto-conversion is applied to
+ replace default values.
  - As mentioned in [Behavioral Changes](#behavioral-changes), selector
- defaulting is disabled.
+ defaults are disabled.
  - The default spec.revisionHistoryLimit for all applicable kinds in the
- apps/v1beta2 group version has set to 10.
+ apps/v1beta2 group version is 10.
  - The default spec.successfulJobsHistoryLimit is 3 and spec.failedJobsHistoryLimit
    is 1 on CronJobs.
 
-### [Workload API (batch)]
-- CronJob has been promoted to `batch/v1beta1` ([#41039](https://github.com/kubernetes/kubernetes/issues/41039), [@soltysh](https://github.com/soltysh)).
-- `batch/v2alpha.CronJob` has been deprecated in favor of `batch/v1beta` and will be removed in future releases.
-- Job has now the ability to set a failure policy using `.spec.backoffLimit`.  The default value for this new field is set to 6. ([#30243](https://github.com/kubernetes/kubernetes/issues/30243), [@clamoriniere1A](https://github.com/clamoriniere1A)).
-- `batch/v2alpha1.ScheduledJob` has been removed.
-- Job controller creates pods in batches instead of all at once ([#49142](https://github.com/kubernetes/kubernetes/pull/49142), [@joelsmith](https://github.com/joelsmith)).
-- Short `.spec.ActiveDeadlineSeconds` is properly applied to a job ([#48545](https://github.com/kubernetes/kubernetes/pull/48454), [@weiwei4](https://github.com/weiwei04)).
+### Workloads API (batch)
```

Comment:
I have given one-line space between the heading and the bullet lists, and between bullet lists